js/src/framework.ts

```diff
@@ -699,7 +703,12 @@ async function runEvaluatorInternal(
           );
           rootSpan.log({ output, metadata });
 
-          const scoringArgs = { ...datum, metadata, output };
+          const scoringArgs = {
```

Comment:
this is a somewhat unrelated change, but it prevents polluting the scorer args with things like `_xact_id`, `id`, etc.